$OpenBSD: patch-src_tests_test_properties_test_properties_cpp,v 1.1.1.1 2014/07/14 09:15:12 zhuk Exp $
1. Unbreak building tests.

2. Upstream commit ac3977c603a5eba581e2e4a557fdee160d3cc615:
https://github.com/mltframework/mlt/commit/ac3977c603a5eba581e2e4a557fdee160d3cc615
Unbreaks running tests.
--- src/tests/test_properties/test_properties.cpp.orig	Sun Jun 29 22:23:17 2014
+++ src/tests/test_properties/test_properties.cpp	Mon Jul 14 09:30:11 2014
@@ -23,7 +23,6 @@
 using namespace Mlt;
 
 extern "C" {
-#define __DARWIN__
 #include <framework/mlt_property.h>
 #include <framework/mlt_animation.h>
 }
@@ -39,6 +38,7 @@ class TestProperties: public QObject (public)
 #if defined(__linux__) || defined(__DARWIN__)
         locale = newlocale( LC_NUMERIC_MASK, "POSIX", NULL );
 #endif
+        Factory::init();
     }
 
     ~TestProperties() {
